pkg/utils/timed_version.go

```diff
@@ -76,6 +82,28 @@ func (t *TimedVersion) After(other *TimedVersion) bool {
 	return t.ts > other.ts
 }
 
+func (t *TimedVersion) Compare(other *TimedVersion) int {
+	t.lock.RLock()
+	defer t.lock.RUnlock()
+
+	if t.ts == other.ts {
+		if t.ticks == other.ticks {
+			return 0
+		}
+		if t.ticks > other.ticks {
+			return 1
+		}
+		return -1
+	}
+	if t.ts == other.ts {
```

Comment:
this is already checked above?

Reply:
that's what i get for copy/pasting. thanks